src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs

```diff
@@ -347,7 +355,7 @@ public bool WidthIsProportional
         /// <summary>
         /// Set this to true if you want a small vibration every time the index changes.
         /// </summary>
-        public bool VibrateOnSelectionChanged { get; set; }
+        public bool VibrateOnSelectionChanged => true;
```

Comment:
Are consumer able to turn it off now?

Reply:
Nope, that was not supposed to be in the PR. Will update